tests/test_ec2/test_spot_fleet.py

```diff
@@ -87,6 +96,7 @@ def test_create_spot_fleet_with_lowest_price():
     len(spot_fleet_requests).should.equal(1)
     spot_fleet_request = spot_fleet_requests[0]
     spot_fleet_request["SpotFleetRequestState"].should.equal("active")
+    spot_fleet_request["Tags"].should.equal({"Key": "test2", "Value": "value2"})
```

Comment:
This is incorrect.  The tags will come back as an array of dicts.

Check the docs: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ec2/client/describe_spot_fleet_requests.html